docs/index.md

```diff
@@ -44,6 +43,9 @@
 * [Creating a Multidomain Design](./cookbook/creating-a-multidomain-design.md)
 * [Working with Multiple Cron Instances](./cookbook/working-with-multiple-cron-instances.md)
 
+## Model
+* [Basics About Model Architecture](./model/introduction-to-model-architecture.md)
```

Comment:
please change all `basics-about-model-architecture` into `introduction-to-model-architecture`

Reply:
also titles